react/styles/index.js

```diff
@@ -0,0 +1 @@
+export * from '@material-ui/core/styles'
```

Comment:
idea: can we add something like:

```
import { makeStyles } from '@material-ui/core/styles'

const exportedMakeStyles = arg => {
  console.warn('deprecated: do not use me anymore')

  return makeStyles(arg)
}
export default exportedMakeStyles

```

Reply:
+ withStyles, useStyles...

Reply:
no need codemod. If we warn

Reply:
+ we need to document into cozy-ui README, how to do a correct import of import { makeStyles } from '@material-ui/core/styles'